Tools-Override/tests.targets

```diff
@@ -55,11 +60,72 @@
 
     <TestCommandLine Condition="'$(Performance)'!='true'">$(TestProgram) $(TestArguments) {XunitTraitOptions}</TestCommandLine>
   </PropertyGroup>
+  
+  <!-- This will generate a version of the dotnet cli that runs with the live
+       build of corefx using the other components gathered from the stable 
+       version consumed from the Tools directory 
+  -->
+  <Target Name="ConstructDotNetExe" Condition="!Exists('$(BinDir)dotnetcli')" >
```

Comment:
So we have a sharedfx with locally built assemblies and test dir only has xunit.console.netcore and the test dll and it's locally built dependencies.